pkg/apis/policy/v1/iampolicy_types.go

```diff
@@ -45,6 +45,8 @@ type Target struct {
 
 // IamPolicySpec defines the desired state of IamPolicy
 type IamPolicySpec struct {
+	// This defaults to ignoring all cluster role binding names that start with "system:"
```

Comment:
I think this comment is supposed to match the `description` in the CRD... I couldn't get the old operator-sdk version installed and generating this CRD anyway, so it might not matter now. Just something to be aware of when we do upgrade that. Operator-sdk v1+ will take the comments in this struct and populate the relevant fields in the CRD.

Reply:
Got it. I didn't think we used the operator-sdk to generate these manifests based on the directory structure of `deploy`. I'll update it though.

Reply:
@JustinKuli you are right about that. I updated the comment above the struct field.

operator-sdk manifest and code generation is currently broken in this repo. The manifest generation fails with this:
```
iam-policy-controller on  allowlist-regex [?] via 🐹 v1.16.8 ❯ ./operator-sdk generate crds     
[Deprecation Notice] Operator SDK has a new CLI and project layout that is aligned with Kubebuilder.
See `operator-sdk init -h` and the following doc on how to scaffold a new project:
https://v0-19-x.sdk.operatorframework.io/docs/golang/quickstart/
To migrate existing projects to the new layout see:
https://sdk.operatorframework.io/docs/building-operators/golang/project_migration_guide/

INFO[0000] Running CRD generator.                       
/home/mprahl/git/iam-policy-controller/pkg/apis/policy/v1/iampolicy_types.go:69:31: map values must be a named type, not *ast.MapType
FATA[0000] error generating CRDs from APIs in pkg/apis: error generating CRD manifests: error running CRD generator: not all generators ran successfully
```
It's failing on this:
https://github.com/open-cluster-management/iam-policy-controller/blob/main/pkg/apis/policy/v1/iampolicy_types.go#L65

Which was introduced over a year ago in https://github.com/open-cluster-management/iam-policy-controller/commit/e06184dca540757c2b10f12e2f72d744c4f4b697.

Reply:
Good to know. And I agree - we can just fix this later when updating the operator-sdk.